Realm/Realm/Linq/RealmResultsVisitor.cs

```diff
@@ -553,14 +553,19 @@ protected override Expression VisitBinary(BinaryExpression node)
                     memberExpression = leftExpression as MemberExpression;
                 }
 
+                if (memberExpression == null)
```

Comment:
We've never encountered this in practice, but technically, there was no code preventing a case where `memberExpression` could be `null`.